src/DiffChecker.ts

```diff
@@ -95,9 +100,7 @@ export class DiffChecker {
       >Object.keys(diffCoverageData)
       for (const key of keys) {
         if (diffCoverageData[key].oldPct !== diffCoverageData[key].newPct) {
-          const oldValue: number = Number(diffCoverageData[key].oldPct)
-          const newValue: number = Number(diffCoverageData[key].newPct)
-          if (oldValue - newValue > delta) {
+          if (this.getPercentageDiff(diffCoverageData[key]) > delta) {
             return true
```

Comment:
Looks like the new function added does (newPct - oldPct) whereas the logic here relied on (oldPct - newPct). 
Since the new function is being used at many places maybe we can use (-this.getPercentageDiff(diffCoverageData[key])) here

Reply:
Makes sense. I missed out on this subtle difference. I'll update it  😄